latex2edx/plastexit.py

```diff
@@ -180,8 +189,8 @@ def filter_fix_abox_with_linenum(m):
     @staticmethod
     def fix_unicode(stxt):
         ucfixset = { u'\u201d': '"',
-                     u'\u2014': '-',
-                     u'\u2013': '-',
+                     u'\u2014': '&#8212;',
+                     u'\u2013': '&#8211;',
```

Comment:
Curious that it was encoded as above and worked fine. Are escaped HTML entities handled differently in edX now? Not sure, but this works well. Thanks, Eric!